aks/scfcharts.tf

```diff
@@ -20,6 +20,36 @@ resource "helm_release" "uaa" {
   values = [
     "${file("${local.chart_values_file}")}"
   ]
+  # scf-config-values
+  set {
+    name = "env.DOMAIN"
+    value = "${var.cap_domain}"
+  }
+  set {
+    name = "env.UAA_HOST"
+    value = "uaa.${var.cap_domain}"
+  }
+  set_sensitive {
+    name = "secrets.CLUSTER_ADMIN_PASSWORD"
+    value = "${var.admin_password}"
+  }
+  set_sensitive {
+    name = "secrets.UAA_ADMIN_CLIENT_SECRET"
+    value = "${var.root_password}"
+  }
+  # stratos-metrics-values
+  set {
+    name = "kubernetes.apiEndpoint"
+    value = "${var.cap_domain}"
+  }
+  set {
+    name = "nginx.username"
+    value = "${var.username}"
+  }
```

Comment:
These params are used only for the metrics chart, not for UAA or scf, or even stratos console. Need to remove.